### ISSUE TITLE
Upgrade to Rails 5

### ISSUE THREAD

**Issue by Nimphal** [CottageLabs/willow#34]
Sufia 7.2 prefers it.

**Comment by Nimphal** [comment]
They may prefer it, but is currently not possible, because:

```
sufia (= 7.2.0) was resolved to 7.2.0, which depends on
      browse-everything (>= 0.10.3) was resolved to 0.10.5, which depends on
        google-api-client (~> 0.8.6) was resolved to 0.8.7, which depends on
          activesupport (< 5.0, >= 3.2)
```

Until browse-everything make a new release (it's in the pipeline) that depends on google-api-client 0.9.x (which does not specify a maximum activesuppport version) sufia 7.2 is not compatible with Rails 5. browse-everything know about the incompatibility, they are looking to do some additional testing before they cut the release.

**Comment by Nimphal** [comment]
This has been done, closing.
